css/about.css

```diff
@@ -0,0 +1,202 @@
+:root {
+    --primary: #CFF4EC;
+    --primary-dark: #94D6C8;
+
+    --secondary: #E2F6CD;
+
+    --neutral: #F2F2F1;
+
+    --text-color: #2F2F2E; 
+
+    --btn-color: #4D7369;
+
+    --white: #ffffff;
+}
+
+body{
+    box-sizing: border-box;
+    font-family: 'Quicksand', sans-serif;
+    color: var(--text-color);
+    margin: 0;
+    padding: 0;
+    overflow-X: hidden;
+}
+
+header {
+    padding-bottom: 20px;
```

Comment:
This isn't getting applied because you've got `p-0` on your header. I don't think you need `p-0`